did_contract/src/lib.rs

```diff
@@ -15,7 +15,8 @@ use option::OptionAddress;
 use soroban_sdk::{contract, contractimpl, panic_with_error, Address, Env, String, Vec};
 use verifiable_credential::VerifiableCredential;
 
-const LIFE_TIME: u32 = 1_578_000;
+const LEDGERS_THRESHOLD: u32 = 1;
+const LEDGERS_TO_LIVE: u32 = 520_000;
```

Comment:
According to my calculations, the 31 days would be 535_680 ledgers. We could leave it at 535_000.